client/components/LoginScreen.js

```diff
@@ -50,7 +74,9 @@ class LoginScreen extends Component {
           <h2>Log in</h2>
           <hr />
           <input name="email" placeholder="Email" onChange={this.handleChange}/>
+          { emError ? <h5 className="noQty">{emError}</h5> : <div /> }
           <input name="password" type="password" placeholder="Password" onChange={this.handleChange}/>
+          { pwError ? <h5 className="noQty">{pwError}</h5> : <div /> }
```

Comment:
Not a bug or anything, but it'd be nice if you cleared out the old error in the emError or pwError section when you go to try again and get a different error.